blaze/utils.py

```diff
@@ -271,6 +282,26 @@ def register(typename, converter, converters=_converters):
 object_hook.register('builtin_function', lambda x: getattr(builtins, x))
 
 
+def numpy_pandas_function_from_str(f):
+    """
+    reconstruct function from string representation
+    """
+    if f.startswith("numpy"):
+        mod = np
+
+    elif f.startswith("pandas"):
+        mod = pd
+
+    else:
+        raise NotImplementedError("accepts numpy/pandas/builtin funcs only")
+
+    fcn = reduce(getattr, f.split('.')[1:], mod)
+    return fcn
+
+
+object_hook.register('numpy_pandas_function', numpy_pandas_function_from_str)
+
+
```

Comment:
Added `json_dumps` and `object_hook` for pandas/numpy functions used in `map`. Added simple tests but need more interesting examples.

@kwmsmith